PBR/src/PBR_Renderer.cpp

```diff
@@ -387,14 +387,17 @@ PBR_Renderer::~PBR_Renderer()
 void PBR_Renderer::PrecomputeBRDF(IDeviceContext* pCtx,
                                   Uint32          NumBRDFSamples)
 {
+
+    bool IsSupportedUnormFormat = m_Device.GetTextureFormatInfo(TEX_FORMAT_RG16_UNORM).Supported;
+
     TextureDesc TexDesc;
     TexDesc.Name            = "Preintegrated GGX";
     TexDesc.Type            = RESOURCE_DIM_TEX_2D;
     TexDesc.Usage           = USAGE_DEFAULT;
     TexDesc.BindFlags       = BIND_SHADER_RESOURCE | BIND_RENDER_TARGET;
     TexDesc.Width           = BRDF_LUT_Dim;
     TexDesc.Height          = BRDF_LUT_Dim;
-    TexDesc.Format          = TEX_FORMAT_RG16_FLOAT;
+    TexDesc.Format          = IsSupportedUnormFormat ? TEX_FORMAT_RG16_UNORM : TEX_FORMAT_RG16_FLOAT;
```

Comment:
RG16F is actually the safest choice.
16-bit UNORM formats are not supported without extension even in GLES3.2, while 16F is fully supported in GLES3.1 and available in GLES3.0
I suggest removing this for now, I will fix the format detection.